qiita_db/test/test_analysis.py

```diff
@@ -95,12 +100,12 @@ def test_create(self):
 
         new = Analysis.create(User("admin@foo.bar"), "newAnalysis",
                               "A New Analysis")
-        self.assertEqual(new.id, 3)
+        self.assertEqual(new.id, 7)
```

Comment:
Oh no, this is something that we are trying to remove, and @wasade has done strict comments on changing this, and I thing all @biocore/qiita-dev agree that this has to be done so we do not spend more time fixing magic numbers...